R/utils.R

```diff
@@ -0,0 +1,15 @@
+## Until the future package provides a well-defined *apply() API
+## we define futurized versions of lapply() and apply() here.
```

Comment:
By the way, there's probably a reason why these are not in `future` yet, but I can't really figure out why? (Just curious, no need to elaborate if the reason is complicated.)